README.md

```diff
@@ -20,7 +20,7 @@ Genomics-Workspace is a bioinformatic website project created by [i5k Workspace]
 * Task queue with [Celery](https://github.com/celery/celery) and [RabbitMQ](http://www.rabbitmq.com/).
 * Use [PostgreSQL](https://www.postgresql.org/) as database backend.
 * BLAST:
-  * generates all BLAST output formats for download: Text, CSV, TSV, XML, ASN.1.
+  * generates serveral BLAST output formats for download: Text, TSV, XML, ASN.1.
```

Comment:
typo: several instead of serveral

Reply:
Thank you for finding this. I have fixed and update this pull request.